.gitignore

```diff
@@ -303,5 +303,5 @@ AutorestSwift.xcodeproj/xcuserdata/
 Pods/
 .build/
 .swiftpm/
-test/integration/generated/body-file/AutoRestSwaggerBatFile.xcodeproj/xcuserdata
-test/integration/generated/head/AutoRestHeadTest.xcodeproj/xcuserdata
+*.plist
```

Comment:
I don't think we can just add plists to .gitignore. Some plists we need checked in because each project looks for an info.plist in its build settings.

Reply:
Thanks. Revert.